x-pack/test/functional/apps/code/explore_repository.ts

```diff
@@ -229,71 +229,71 @@ export default function exploreRepositoryFunctonalTests({
       });
 
       // TODO(qianliang): blocked by https://github.com/elastic/code/issues/1163
-      // it('Click file/directory on the right panel', async () => {
-      //   log.debug('Click file/directory on the right panel');
-
-      //   // Wait the file tree to be rendered and click the 'src' folder on the file tree.
-      //   await retry.try(async () => {
-      //     expect(await testSubjects.exists('codeFileExplorerNode-src')).to.be(true);
-      //   });
-
-      //   await testSubjects.click('codeFileExplorerNode-src');
-      //   await retry.try(async () => {
-      //     expect(await testSubjects.exists('codeFileExplorerNode-models')).to.be(true);
-      //   });
-
-      //   await testSubjects.click('codeFileExplorerNode-models');
-      //   // Then the 'models' folder on the file tree.
-      //   await retry.try(async () => {
-      //     expect(await testSubjects.exists('codeFileExplorerNode-User.ts')).to.be(true);
-      //   });
-
-      //   await testSubjects.click('codeFileExplorerNode-User.ts');
-      //   // Then the 'User.ts' file on the file tree.
-      //   await retry.try(async () => {
-      //     expect(await testSubjects.exists('codeSourceViewer')).to.be(true);
-      //   });
-      // });
+      it('Click file/directory on the right panel', async () => {
```

Comment:
I think in this PR, the explore-repository test is skipped, so do we need to keep the changes in this file?

Reply:
Let's keep the change since we are gonna uncomment this any way